### PR TITLE
pint: 0.28.3

### PR DESCRIPTION
**Pull request by robsonpeixoto**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/pint.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  homepage "https://cloudflare.github.io/pint/"
+  url "https://github.com/cloudflare/pint/archive/refs/tags/v0.28.3.tar.gz"
+  sha256 "5c12d379a08de6483e8c7b7503c3b317cc639c8ecbd8015b8d415ec98018328a"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X 'main.version=#{version}' -X 'main.commit=homebrew'"), "./cmd/pint"
+  end
```
Comment (carlocab) [inline Formula/pint.rb:12]:
```suggestion
  def install
    ldflags = %W[
      -s -w
      -X main.version=#{version}
      -X main.commit=#{tap.user}
    ]
    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/pint"
  end
```

File: Formula/pint.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          expr: job:request_latency_seconds:mean5m{job="myjob"} > 0.5
+          for: 10m
+          labels:
+            severity: page
+          annotations:
+            summary: High request latency
+    EOS
+
+    assert_equal("level=info msg=\"File parsed\" path=#{testpath}/test.yaml rules=1\n",
+      shell_output("#{bin}/pint -n lint #{testpath}/test.yaml 2>&1"))
```
Comment (carlocab) [inline Formula/pint.rb:29]:
```suggestion
    assert_equal("level=info msg=\"File parsed\" path=#{testpath}/test.yaml rules=1\n",
                 shell_output("#{bin}/pint -n lint #{testpath}/test.yaml 2>&1"))
```

**Comment by robsonpeixoto** [comment]
Done! Thanks @carlocab

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**